language_detector/main.py

```diff
@@ -2,10 +2,31 @@
 
 """This is the entry point of the program."""
 
-from languages import LANGUAGES
+from .languages import LANGUAGES
 
 
 def detect_language(text, languages=LANGUAGES):
     """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    
+    ECounter = 0
+    SpanCounter = 0
+    GermCounter = 0
```

Comment:
Other than Kekoa's comment about the use of a dynamic dictionary, the code looks good! It is very similar to our program.  Good stuff!!